### DISCUSSION TITLE
[Q&A] I would like to inquire whether GLPI can integrate with Okta for SSO?

### DISCUSSION THREAD

**Discussion by psl2024** [glpi-project/glpi-agent#759] [Q&A]
### Documentation & bug reporting acknowledgment

Yes, I read it

### Describe your problem

Hi Sirs

Good day!

I am trying to integrate our GLPI with Okta. I installed the glpisaml plugin and configured it according to the sample documentation, but it is not working. Is there a way to get more comprehensive documentation?
![image](https://github.com/user-attachments/assets/8b929d5e-447e-4985-9f6e-76da2a10a013)

Thank you in advance!

**Comment by g-bougard** [comment]
Hi @psl2024 

not sure this is the right place to discuss about glpi server only concerns. You should try glpi project forum first if you didn't tried it.

**Reply by psl2024** [reply]
I will take note on this. Thank you @g-bougard
